src/login/components/GroupManagement/Groups/EditGroup.js

```diff
@@ -18,22 +21,45 @@ const EditGroup = (props) => {
           save
         </button>
       </div>
-      <nav>
-        <li>
-          <p>Invites</p>
-        </li>
-        <li>
-          <p>People</p>
-        </li>
-        <li>
-          <p>Delete</p>
-        </li>
-      </nav>
-      <InvitesParent group={group} />
+      <EditGroupNav parentId={parentId} setNavParent={setNavParent} />
```

Comment:
`setNavParent` does not to do anything so I think it is best remove it.

Reply:
This function picks up the id from the nav `<li>` and sets it to state. If I remove it from here it won't run in EditGroupNav. If you think an alternative implementation is better please share more details on what you're thinking.